src/blacklist.js

```diff
@@ -0,0 +1,12 @@
+const Url = require("url");
+
+const blacklist = {
+  'www.google-analytics.com': true,
+  'api.mapbox.com': true,
+  'a.tiles.mapbox.com': true,
+  'b.tiles.mapbox.com': true,
+}
+
+const isInBlacklist = url => blacklist[Url.parse(url).hostname]
```

Comment:
I don't feel comfortable with this. It feels too "brutal". I don't even know what `api.mapbox.com` is but imagine it's a REST endpoint that gives the web page the JSON it needs to paint the screen in a certain way onload. Then blacklisting it would prevent the DOM becoming what it becomes for normal browser users. 

First of all, let's minimize the list of default domains in the list. Second, I wonder if we should do more to filter than just look at the domain. What if you want *some* assets from a certain domain but skip all others. For example, you might to blacklist `a.tiles.mapbox.com/*.js`.

Reply:
The name isn't ideal. That fact that it's a list internally is not important externally. The function can be called `isBlacklisted` or `blacklistedUrl` or perhaps even more user friendly is `skippableUrl`. What do you think? To me the word "blacklisted" means something very permanent. Like things that are set in stone and unlikely to change other than more potentially being added.

Reply:
1. Yes agree `api.mapbox.com` is too specific. 
2. `a.tiles.mapbox.com` provides tiles which are either images or binary vector files (protobuf file format). It would make sense to blacklist `*.tiles.mapbox.com`